DiceKeys/Views/Screens/CustomeDiceDemo/BlankView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BlankView.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 16/01/21.
+//
+
+import SwiftUI
+
+/// BlankViewModel
+class BlankViewModel: ObservableObject {
```

Comment:
There shouldn't need to be a blank view if the other view has a model that allows letter, digit, and orientation to be nil.

Reply:
Absolutely, did the refactoring of code accordingly.